src/app/components/connection-window/connection-window.component.ts

```diff
@@ -14,11 +16,13 @@ import { SubscriptionHandler } from '../../utils/subscription-handler/subscripti
 export class ConnectionWindowComponent extends SubscriptionHandler implements OnInit {
 
   public readonly KEIRA_VERSION = version;
+  private configStorage: Config = JSON.parse(localStorage.getItem('config'));
```

Comment:
what if `config` is null or undefined? I'd rather move this logic inside a method (that is then called by `ngOnInit`)